utils/sql-db-seeder/src/utilisation-report/utilisation-report.helper.ts

```diff
@@ -78,8 +78,8 @@ export const createUploadedReport = (
   utilisationReport.reportPeriod = reportPeriod;
   utilisationReport.status = status;
   utilisationReport.dateUploaded = new Date();
-  utilisationReport.uploadedByUserId = null;
-  utilisationReport.updatedByUserId = getDbAuditUpdatedByUserId({ platform: 'TFM', userId: portalUser._id.toString() });
+  utilisationReport.uploadedByUserId = portalUser._id.toString();
+  utilisationReport.updatedByUserId = getDbAuditUpdatedByUserId({ platform: 'PORTAL', userId: portalUser._id.toString() });
```

Comment:
Noticed from demo that this seed helper generated incorrect data for the uploaded report, so decided to update it here

Reply:
Nice, thanks!